Kickstarter-iOS/Views/Controllers/ChangePasswordViewController.swift

```diff
@@ -211,6 +211,13 @@ final class ChangePasswordViewController: UIViewController, MessageBannerViewCon
     }
   }
 
+  private func dismissKeyboard() {
+    _ = [self.newPasswordTextField, self.confirmNewPasswordTextField, self.currentPasswordTextField]
```

Comment:
Would it be a problem to just blanket ping all of the textFields to resign?

```swift
[self.newPasswordTextField, self.confirmNewPasswordTextField, self.currentPasswordTextField]
  .forEach { $0.resignFirstResponder() }
```
Also `resignFirstResponder()` returns `Void` so no need for the `_ =` to discard the return value.

Reply:
Not at all. I can change that.